docs/metrics/v3.md

```diff
@@ -343,3 +343,13 @@ The standard metrics group for GoCollector is not shown below.
 | `minio_replication_max_queued_bytes`              | `gauge` | Maximum number of bytes queued for replication since server start           | `server` |
 | `minio_replication_max_queued_count`              | `gauge` | Maximum number of objects queued for replication since server start         | `server` |
 | `minio_replication_max_data_transfer_rate`        | `gauge` | Maximum replication data transfer rate in bytes/sec seen since server start | `server` |
+
+### `/ilm`
+
+| Name                                                  | Type      | Help                                                                       | Labels |
+|-------------------------------------------------------|-----------|----------------------------------------------------------------------------|--------|
+| `minio_cluster_ilm_expiry_pending_tasks`              | `counter` | Number of pending ILM expiry tasks in the queue                            |        |
+| `minio_cluster_ilm_transition_active_tasks`           | `counter` | Number of active ILM transition tasks                                      |        |
+| `minio_cluster_ilm_transition_missed_immediate_tasks` | `counter` | Number of missed immediate ILM transition tasks                            |        |
+| `minio_cluster_ilm_transition_pending_tasks`          | `counter` | Number of pending ILM transition tasks in the queue                        |        |
+| `minio_cluster_ilm_versions_scanned`                  | `counter` | Total number of object versions checked for ilm actions since server start |        |
```

Comment:
Need to add the `server` label to these metrics now

Reply:
done